downstream/assemblies/platform/assembly-upgrading-to-aap.adoc

```diff
@@ -16,7 +16,11 @@ ifdef::context[:parent-context: {context}]
 
 To upgrade your {PlatformName} to the later version, download the desired version of the {PlatformNameShort} installer. Configure it with the same parameters you have used in previous installations, configure any new parameters, then run the installer.
 
-The following guide describes these steps in more detail.
+In some cases, you may need to configure other settings if you plan on upgrading other components of your {PlatformNameShort}, such as {ControllerName}. The following sections describes these steps in further detail.
+
+include::platform/con-upgrade-planning.adoc[leveloffset=+1]
+
+Once you have necessary changes during upgrade planning, proceed by downloading the desired version of the {PlatformNameShort} installer.
```

Comment:
Suggestion: `Once you have necessary changes during upgrade planning` > `Once you have completed the necessary changes during upgrade planning`